PowerPointLabs/PowerPointLabs/ColorsLab/ColorsLabPaneWPF.xaml.cs

```diff
@@ -94,6 +94,22 @@ public void LoadFavoriteColorsFromPath(string filePath)
             dataSource.LoadThemeColorsFromFile(filePath);
         }
 
+        public List<Color> GetRecentColorsPanelAsList()
+        {
+            List<HSLColor> recentHslColors = dataSource.GetListOfRecentColors();
+            List<Color> recentColors = new List<Color>();
```

Comment:
Would you be able to do `List<Color> recentColors = new List<Color>(recentHslColors);` ?

Reply:
Unfortunately no... Because `dataSource.GetListOfRecentColors()` actually returns a `List<HSLColor>`, and if I were to do `new List<Color>(someList)`, `someList` has to be a `List<Color>` else it wouldn't work. It's not able to convert directly between two different `Lists`. So this is already the easier way to do the conversion. 

It also wouldn't help to change `dataSource.GetListOfRecentColors()` to return a `List<Color>` directly, because there are other places that calls this method and works with the `HSLColor`, so wouldn't make sense to change.